### ISSUE TITLE
tap_action > entity not used for more-info action

### ISSUE THREAD

**Issue by crowbarz** [jcwillox/lovelace-paper-buttons-row#2]
I am trying to specify an alternate entity to show more-info for on a button tap. I believe the following card config should bring up the more-info panel for `remote.living_kitchen_harmony` on a single tap, but for me it shows more-info for the underlying button's entity (`script.all_off_av` in this case.)

```
entities:
  - align_icons: bottom
    buttons:
      - align_icon: top
        entity: script.all_off_av
        icon: 'mdi:power-off'
        name: All Off
        tap_action:
          action: more-info
          entity: remote.living_kitchen_harmony
        hold_action:
          action: call-service
          service: script.all_off_av
    type: 'custom:paper-buttons-row'
type: entities
```

Using paper-buttons-row v0.1.1 via HACS 0.23.2 on HA Core 105.5. I don't think I have any other plugins installed that might conflict. (I have uninstalled button-entity-row: your rewrite solved many of the limitations that I encountered using that plugin ... thanks!)

**Comment by jcwillox** [comment]
Thanks, this is definitely a bug. I've fixed it and will release the patch within the next hour 👍.

**Comment by jcwillox** [comment]
Fixed in release [`0.1.2`](https://github.com/jcwillox/lovelace-paper-buttons-row/releases/tag/0.1.2).

**Comment by TobiasSchwarz74** [comment]
Still not working in latest release 1.0.0
